swagger-v1/swagger-unresolved.yaml

```diff
@@ -1795,7 +1796,8 @@ components:
     PaymentGatewayRaftCredentials:
       required:
       - credentialType
-      - license
+      - stpTerminalId
+      - bankId
```

Comment:
bankId needs to be changed to stpBankId or else it won't show up as a required field aka it's missing the star next to it